core/services/DicomSvc.cc

```
@@ -0,0 +1,675 @@
+ * @return Pair of jaw positions for the specified side.
+ */
+std::pair<double,double> IDicomPlan::ReadJawsAperture(const std::string& planFile,const std::string& side,int beamIdx, int controlpointIdx){
+  if(side=="X"){
+    auto x1 = ReadJawPossition(planFile,"X1",beamIdx,controlpointIdx);
+    auto x2 = ReadJawPossition(planFile,"X2",beamIdx,controlpointIdx);
+    return std::make_pair(x1,x2);
+  }else if(side=="Y"){
+    auto y1 = ReadJawPossition(planFile,"Y1",beamIdx,controlpointIdx);
+    auto y2 = ReadJawPossition(planFile,"Y2",beamIdx,controlpointIdx);
+    return std::make_pair(y1,y2);
+  }
```

Cppcheck (2.19.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)